pytest_fixtures/satellite_auth.py

```diff
@@ -42,6 +42,18 @@ def ipa_data():
     }
 
 
+@fixture(scope='session')
+def open_ldap_data():
+    return {
+        'ldap_user_name': settings.open_ldap.username,
+        'ldap_user_passwd': settings.open_ldap.password,
+        'base_dn': settings.open_ldap.base_dn,
+        'group_base_dn': settings.open_ldap.group_base_dn,
+        'ldap_hostname': settings.open_ldap.hostname,
+        'open_ldap_user': settings.open_ldap.open_ldap_user,
+    }
```

Comment:
I think this is a bit overkill to wrap the settings in a fixture, but if it is something others want to stick with, it would be better to do it like this.
```suggestion
    return settings.open_ldap
```